cosm-lab.site/ru/index.html

```diff
@@ -1,13 +1,13 @@
 <html>
 
-    <head>
-        <title>
-            COSM :: Лаборатория Облачных Решений и Моделирования
```

Comment:
Looks like there are lots of changes related only to indentation. It is better to avoid such changes, because they are hiding real meaningful code changes and make it harder to perform a review

Reply:
Fixed, thanks.